pom.xml

```diff
@@ -1,85 +1,97 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.wojtechm</groupId>
-    <artifactId>SpringJavaConfig</artifactId>
-    <version>0.1</version>
-    <name>SpringJavaConfig</name>
-    <description>SpringJavaConfig</description>
-    <packaging>war</packaging>
+  <groupId>com.github.wojtechm</groupId>
```

Comment:
Konfig wejdzie przed tym kompletem zmian, tam się zmieniło sporo, proponuję poczekać i przebazować się po ich zmianach. Bo oni mają zmian naście już zrobionych a kolejne w trakcie. I nie zawadziłby mi opis bym wiedział na co zwracać uwagę podczas recenzowania 84 zmienionych plików...

Reply:
M.in. zmieniła się grupa, stąd tu uwaga. Proponuję poczytać tamtą recenzję, zobaczyć uwagi, dostosować kod (bym nie musiał ich tu powtórzyć). :-)

Reply:
Heja, właśnie scaliłem zmiany Wiktora i chłopaków. Przejrzyj proszę ten kod pod ich kątem i usprawnij.